### PR TITLE
Adding Online evaluation dependencies in eval environment

### PR DESCRIPTION
**Pull request by ghyadav**

### CODE REVIEW

File: assets/evaluation_on_cloud/environments/evaluations-built-in/context/Dockerfile [hunk 1]
```diff
@@ -5,4 +5,5 @@ RUN pip install -r /app/requirements.txt
 
 # Copy your Python file into the image
 COPY evaluate_on_data.py /app/evaluate_on_data.py
-COPY save_evaluation.py /app/save_evaluation.py
\ No newline at end of file
+COPY save_evaluation.py /app/save_evaluation.py
+ADD online_eval /app/online_eval
```
Comment (vizhur) [inline assets/evaluation_on_cloud/environments/evaluations-built-in/context/Dockerfile:9]: COPY?

File: assets/evaluation_on_cloud/environments/evaluations-built-in/context/requirements.txt [hunk 2]
```diff
@@ -2,4 +2,13 @@ azure-ai-evaluation
 openai
 azureml-mlflow
 azure-identity
-azure-ai-ml
\ No newline at end of file
+azure-ai-ml
+azureml-core
```
Comment (vizhur) [inline assets/evaluation_on_cloud/environments/evaluations-built-in/context/requirements.txt:6]: SDK team advised that it is not recommended to combine v1 and v2 SDK. Please use either v1 or v2.

Please also pin the versions and us templates for AzureML SDK dependnencies

**Review by vizhur — CHANGES_REQUESTED** [review]

**Review by ShipraJain01 — APPROVED** [review]

File: assets/evaluation_on_cloud/environments/evaluations-built-in/context/Dockerfile (hunk 1 above)
Comment (vizhur) [reply assets/evaluation_on_cloud/environments/evaluations-built-in/context/Dockerfile:9]: wondering why add instead of copy

**Review by vizhur — DISMISSED** [review]

File: assets/evaluation_on_cloud/environments/evaluations-built-in/context/Dockerfile (hunk 1 above)
Comment (ghyadav) [reply assets/evaluation_on_cloud/environments/evaluations-built-in/context/Dockerfile:9]: Add is needed to copy the entire directory

**Review by ShipraJain01 — APPROVED** [review]

**Review by shrutiyer — APPROVED** [review]

**Review by vizhur — APPROVED** [review]

**Pull request merged**